pilot/pkg/bootstrap/server.go

```diff
@@ -122,7 +122,6 @@ type Server struct {
 	secureGrpcServer *grpc.Server   //
 	mux              *http.ServeMux // debug
 	httpsMux         *http.ServeMux // webhooks
-	kubeRegistry     *kubecontroller.Controller
```

Comment:
Oh I need this in the other PR because we add the instancehandler from service entry into kube.

Reply:
Yes, will rebase after your pr.